tucson-teststand/calibration_lab/setup_dev_electrometer.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Source this file when starting the container to set it up
+echo "Running setup for environment for ts_electrometer. The index {$CSC_INDEX} is defined in the docker-compose file."
+source /home/saluser/.setup_salobj.sh
+setup ts_config_atcalsys -t current
+setup ts_idl -t current
+setup ts_sal -t current
+setup ts_salobj -t current
+setup ts_xml -t current
+
+echo "# Running custom setups of repos"
+setup -j ts_electrometer -r ~/develop/ts_electrometer
+setup -j ts_config_ocs -r ~/develop/ts_config_ocs
+
+echo '# Installing dependencies and declaring OSPL_URI'
+pip install pyserial  # conda takes too long
```

Comment:
You may want to consider using mamba instead of conda. Tiago and I switched to that and it is MUCH faster in resolving dependencies than conda. Having said that, using pip is fine as well of course.